src/site/content/en/progressive-web-apps/customize-install/index.md

```diff
@@ -69,6 +69,7 @@ event, and update your user interface to indicate that the user can install
 your PWA. This is highlighted below.
 
 ```js
+// Create deferredPrompt for use later to show browser install prompt.
```

Comment:
I don't follow.

Reply:
the explanation is in the paragraph before, I just added a comment here to make it more clear. also tweaked it to 'initialize'